index.html

```diff
@@ -17,17 +17,19 @@
 <body>
     <header>
         <nav>
-            <img src="assets/microsoft-logo.png" alt="Microsoft Logo">
+            <img id="microsoft-logo" src="assets/microsoft-logo.png" alt="Microsoft Logo">
 
             <div class="navbar1">
```

Comment:
Entendo que aqui podemos trabalhar com um nome mais intuitivo não acho um boa pratica escrever números para referenciar uma class ou id podemos trocar para `#navbar-left e #navbar-right` assim qualquer pessoa que for trabalhar no seu código só de olhar entende o estilo mesmo sem ter visto o site!